flask_sqlalchemy/__init__.py

```diff
@@ -663,9 +663,8 @@ class Model(object):
     query = None
 
     def __repr__(self):
-        pks = inspect(self).identity_key[1]
-        pretty_pk = pks[0] if len(pks) == 1 else ', '.join(map(str, pks))
-        return '<{0}({1})>'.format(type(self).__name__, pretty_pk)
+        pk = ', '.join(str(value) for value in inspect(self).identity)
```

Comment:
let's test the `str(value)` with a PK that contains non-ascii characters. such a PK is pretty 💩 but you never know...

Reply:
Yep, that fails. Have a suggestion how to rewrite this for Unicode on Py2? Will need to dig into Flask's compat stuff, maybe there's something there.

Reply:
`repr()` instead of `str()`? slightly uglier in py2 because of the `u` prefix, but pretty much all PKs are numeric anyway